File: src/main/java/net/adventuregame/models/TexturedModel.java
package net.adventuregame.models;

import com.chidozie.core.textures.ModelTexture;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

public class TexturedModel {

    private static final Logger log = LoggerFactory.getLogger(TexturedModel.class);
    private RawModel rawModel;
    private ModelTexture texture;

    public TexturedModel(RawModel rawModel, ModelTexture texture) {
        this.rawModel = rawModel;
        this.texture = texture;
        log.info("Created Textured Model With");
    }

    public RawModel getRawModel() {
        return rawModel;
    }

    public ModelTexture getTexture() {
        return texture;
    }
}

File: src/main/kotlin/net/adventuregame/models/TexturedModel.kt
package net.adventuregame.models

import com.chidozie.core.textures.ModelTexture
import org.slf4j.Logger
import org.slf4j.LoggerFactory

class TexturedModel(val rawModel: RawModel?, val texture: ModelTexture?) {
    init {
        log.info("Created Textured Model With")
    }

    companion object {
        private val log: Logger = LoggerFactory.getLogger(TexturedModel::class.java)
    }
}
